docs/workflows/pypi.rst

```diff
@@ -0,0 +1,70 @@
+.. _pypi-workflow:
+
+Setup your own PyPI:
+====================
+
+This section guides you through the quickest way to to setup ``pulp_python`` to act as your very own
+private ``PyPI``.
+
+Create a Repository:
+--------------------
+
+Repositories are the base objects ``pulp`` uses to store and organize its content. They are automatically
```

Comment:
If this is referring to the Pulp project, it should probably be capitalized, if the cli command, then this is fine.